html-forms/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Form</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <label for="email-input">Email:</label><br />
```

Comment:
instead of using the `br` tag, I would recommend wrapping the label with a div, something like this: 
```html
<div>
  <label for="email-address">Email</label>
  <div>
    <input name="email" type="email" id="email-address">
  </div>
</div>
```

The reason is that the `br` tag is for specific formatting of text, like for poetry, and not for layout formatting.